src/utils.jl

```diff
@@ -126,6 +126,11 @@ function _jump_value(input::JuMP.ConstraintRef)
     return JuMP.dual(input)
 end
 
+function _jump_value(int::Int64)
```

Comment:
is this for testing it should go in the test utils folder 
https://github.com/NREL/PowerSimulations.jl/tree/master/test/test_utils